introducing-java/src/test/java/com/onto/javacourse/intro/findingArrayMedian/FindingArrayMedianTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FindingArrayMedianTest {
+    @Test
+    void testFindMedianPosition() {
```

Comment:
One test case is not enough at all here. Empty array, index is at start, index is at end and etc.